extension/src-diagram-snippets/css/theme.css

```diff
@@ -0,0 +1,30 @@
+/*
+ * KIELER - Kiel Integrated Environment for Layout Eclipse RichClient
+ *
+ * http://rtsys.informatik.uni-kiel.de/kieler
```

Comment:
I wonder whether this line shouldn't rather be updated to https://github.com/kieler, since that is now our de facto home page of the kieler project